hw2/04-background.js

```diff
@@ -0,0 +1,38 @@
+// Add your code here
+const Button = document.querySelector('button');
+const Input = document.querySelector('input');
+const Body = document.querySelector('body');
+let IntervalId;
+Body.style.backgroundColor = '#7160d9';
+
+function setBackgroundColor() {
+  const val1 = Math.random() * 255;
+  const val2 = Math.random() * 255;
+  const val3 = Math.random() * 255;
```

Comment:
wouldn't you want to have this surrounded by a math.floor? math.random() will return any number and it is then multiplied by 255.

Reply:
I don't this this would be required. The formula to get a random number between a min and a max is: Math.random() * (max - min) + min. This would be the equivalent of Math.random() * (255 - 0) + 0 => Math.random() * 255. 

Math.Random returns a number between 0 and 1 without multiplying it by anything. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random